benchmarks/benchmark_latency.py

```diff
@@ -115,6 +116,11 @@ def run_to_completion(profile_dir: Optional[str] = None):
     parser.add_argument('--enforce-eager',
                         action='store_true',
                         help='enforce eager mode and disable CUDA graph')
+    parser.add_argument('--kv-cache-dtype',
+                        type=str,
+                        choices=['fp8', None],
```

Comment:
In general, please be explicit in this PR that the `fp8` means `fp8_e5m2`. Given that there are multiple ways to implement `fp8`, this will make things more clear.
```suggestion
                        choices=['fp8_e5m2', None],
```

Reply:
done